src/components/application_manager/src/application_impl.cc

```diff
@@ -755,6 +756,10 @@ UsageStatistics& ApplicationImpl::usage_report() {
   return usage_report_;
 }
 
+HelpPromptManager& ApplicationImpl::help_prompt_manager() {
```

Comment:
@SNiukalov Make this method const, please

Reply:
No using not constant method from object